runtime/Cargo.toml

```diff
@@ -114,5 +118,6 @@ runtime-benchmarks = [
 	"pallet-treasury/runtime-benchmarks",
 	"pallet-vesting/runtime-benchmarks",
 	"pallet-utility/runtime-benchmarks",
+	#"pallet-teeracle/runtime-benchmarks",
```

Comment:
Should this pallet not be benchmarked?

Reply:
Yes. I will comment it out in the next task. https://github.com/integritee-network/integritee-node/issues/65